packages/corvid-e2e-tests/test/drivers/cliDriver.js

```diff
@@ -53,10 +56,14 @@ module.exports = ({ cwd }) => {
   const openEditor = async ({ remoteDebuggingPort, env } = {}) =>
     withCommand("open-editor", { remoteDebuggingPort, env });
 
+  const killAll = async (signal, options) =>
```

Comment:
Was this needed because of a specific bug or just to be safe?
(I think it's probably a good idea either way)

Reply:
during investigations, i thought we might have hanging processes.. not sure it was the case, but as you said, i think it's a good idea anyway